util/network-devp2p/src/host.rs

```diff
@@ -566,7 +574,7 @@ impl Host {
 			self.connect_peer(&id, io);
 			started += 1;
 		}
-		debug!(target: "network", "Connecting peers: {} sessions, {} pending, {} started", self.session_count(), self.handshake_count(), started);
+		debug!(target: "network", "Connecting peers: {} sessions, {} pending, {} started", egress_count + ingress_count, self.handshake_count(), started);
```

Comment:
Why not `handshake_count`? Calling a method initiates another iteration of `session_count()`.